docs/resources/client/level-components/capacity-string.md

```diff
@@ -0,0 +1,157 @@
+| 33 | EffectBatchNodeBot2 | Animated, zLayer B2 | -13 |
+| 34 | EffectBatchNodeAddBot2 | Animated, Blending, zLayer B2 | -14 |
+| 35 | EffectBatchNodeBot3 | Animated, zLayer B3 | -20 |
+| 36 | EffectBatchNodeAddBot3 | Animated, Blending, zLayer B3 | -21 |
+| 37 | EffectBatchNodeBot4 | Animated, zLayer B4 | -27 |
+| 38 | EffectBatchNodeAddBot4 | Animated, Blending, zLayer B4 | -28 |
+| 39 | BatchNodeTextTop3 | Text, zLayer T3 | 21 |
+| 40 | BatchNodeTextAddTop3 | Text, Blending, zLayer T3 | 22 |
+| 41 | BatchNodeTextTop2 | Text, zLayer T2 | 13 |
+| 42 | BatchNodeTextAddTop2 | Text, Blending, zLayer T2 | 14 |
+| 43 | BatchNodeTextTop1 | Text, zLayer T1 | 6 |
+| 44 | BatchNodeTextAddTop1 | Text, Blending, zLayer T1 | 7 |
+| 45 | BatchNodeText | Text, zLayer B1 | -5 |
+| 46 | BatchNodeAddText | Text, Blending, zLayer B1 | -4 |
+| 47 | BatchNodeTextBot2 | Text, zLayer B2 | -12 |
+| 48 | BatchNodeAddTextBot2 | Text, Blending, zLayer B2 | -11 |
+| 49 | BatchNodeTextBot3 | Text, zLayer B3 | -19 |
+| 50 | BatchNodeAddTextBot3 | Text, Blending, zLayer B3 | -18 |
+| 51 | BatchNodeTextBot4 | Text, zLayer B4 | -26 |
+| 52 | BatchNodeAddTextBot5 | Text, Blending, zLayer B4 | -25 |
+| 53 | EffectBatchNodeAddTop4 | Animated, Blending, zLayer T4 | 20 |
+
+## Legacy String
+
+There are other types of Capacity Strings from older versions of the game. The total number of Nodes is used to identify them
+
+| Update | Node count |
+|:-------|:-----------|
+| 1.9 | 4 |
+| 2.0 | 16 |
+
+The Capacity String in 2.0 is quite easy to understand as it is the first 16 BatchNodes of 2.1's Capacity String are exactly the same. 1.9 is quite different with certain index's containing mulitple batchNodes
+ 
+> **Below is a table for 1.9's Capacity String.**  
+> Refer to the [Structure](/resources/client/level-components/Capacity-string?id=structure) section for the properties of the batchNodes
+
+| Index | BatchNode(s) |
+|:------|:-------------|
+| 0 | BatchNode |
+| 1 | BatchNodeAdd</br> BatchNodeAddGlow |
+| 2 | BatchNodeBottom</br> BatchNodeBottom2 |
+| 3 | BatchNodeAddBottom</br> BatchNodeAddBottom2</br> BatchNodeAddBottomGlow</br> BatchNodeAddBottom2Glow |
+
+## High Capacity Mode
+
+In the game options, there is a option the player can toggle called `High Capacity Mode`. when enabled, the Capacity for each BatchNode is set to a hardcoded value. There are 3 capacity sizes
+
+| size | Batch Type |
+|:-----|:-----------|
+| 999 | BatchNode |
+| 200 | EffectBatchNode |
+| 100 | BatchNodeText |
+
+If enabled, the Capacity string is disregarded and these values are used instead.
+
+
+
+## Capacity String Generation
+
+The Capacity String is only generated during verification of the level. When playing the level, the game stores the the highest number of GameObjects that the Batch is responsible for inside its textureAtlas as the capacity. After the level is verified, the following function is called
+
+```cpp
+int capacity = BatchNode->getAtlasCapacity();
```

Comment:
capital variable/member names :disappointed:
honestly we should probably establish a shared code style for code examples since i'm not very convinced by the looks of this style